### ISSUE TITLE
about train

### ISSUE THREAD

**Issue by catofhost** [GaryZhu1996/GARNet#2]
Hey expert, could you please guide me on how to set up the training of GarNet according to the paper? The paper states: "train our multi - view reconstruction network with a batch size of 32 for 200 epochs, with 140 epochs using only BCE loss function and then 60 epochs using the complete loss function as mentioned previously." When I train the single - view model, the learning rate drops to zero around 140 epochs, and the IoU is 0.66. Can you give me some guidance? Thank you so much!
